timemachine/fe/topology.py

```diff
@@ -629,8 +629,8 @@ def parameterize_nonbonded(self, ff_q_params, ff_lj_params):
             ]
         ).astype(np.float64)
 
-        combined_lambda_plane_idxs = None
-        combined_lambda_offset_idxs = None
+        combined_lambda_plane_idxs = np.zeros(NA + NB, dtype=np.int32)
```

Comment:
The potential is effectively broken without this. Because we only used it parameterize other nonbonded potentials we hadn't seen an error.